chia/wallet/puzzles/p2_conditions_curryable.clsp

```diff
@@ -0,0 +1,3 @@
+(mod (CONDITIONS_LIST)
```

Comment:
I think it's slightly dump optimization to just make this `(mod CONDITIONS_LIST` so the compiled output is just `01` but obviously that's a very standard puzzle so this file is certainly not waranted.  I'm not sure why this is necessary, I think the currying is just overhead here instead of just `(q . conditions)`

Reply:
This file wasn't even being used for anything, so have deleted it